### PR TITLE
fix-flutter-dart-code-quality

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

File: templates/flutter/lib/package.dart.twig [hunk 1]
```diff
@@ -1,7 +1,24 @@
+library {{ language.params.packageName }};
+
+
```
Comment (eldadfux) [inline templates/flutter/lib/package.dart.twig:3]: Extra space, or is this a Dart convention?

File: templates/flutter/lib/package.dart.twig [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+import 'package:dio/dio.dart';
+import 'package:meta/meta.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter_web_auth/flutter_web_auth.dart';
+import 'package:dio/adapter.dart';
+import 'package:dio_cookie_manager/dio_cookie_manager.dart';
+import 'package:cookie_jar/cookie_jar.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
```
Comment (eldadfux) [inline templates/flutter/lib/package.dart.twig:15]: Extra space, or is this a Dart convention?

**Comment by eldadfux** [comment]
Added minor comments, let me know when we can merge.

File: templates/flutter/lib/package.dart.twig (hunk 2 above)
Comment (lohanidamodar) [reply templates/flutter/lib/package.dart.twig:15]: signle space is convention, more than one, that's extra, i'll deal with

**Comment by lohanidamodar** [comment]
@eldadfux good to merge

**Review by eldadfux — APPROVED** [review]

**Pull request merged**